tests/e2e/specs/auth-flow-editor.test.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activatePlugin,
+	loginUser,
+	createURL,
+	visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	setAuthToken,
+	setClientConfig,
+	setSearchConsoleProperty,
+	setSiteVerification,
+} from '../utils';
+
+function requestHandler( request ) {
```

Comment:
Since we have quite a few request interception utilities throughout the tests now, maybe we can make these more DRY?

Reply:
I agree, I'm not quite sure what the best way forward is yet but I have been thinking about it.

Reply:
Perhaps a small utility function that accepts the URL to look for (as a regex) and the response object to send?

That function would then call `await page.setRequestInterception( true );` and run the whole `page.on( 'request', request => { ... }` logic.

Reply:
That sounds like a good start 👍 I'll see what I can do in a few minutes.